### DISCUSSION TITLE
cover images / social cards

### DISCUSSION THREAD

**Discussion by StefanoCecere** [vuepress-theme-hope/vuepress-theme-hope#4104] [Q&A] [answered]
i found in the demo a page with a 'cover' metafield. but it's not documented. and seems to work only with images inside the .vuepress folder. 

these images could be used also as images for links?

**Comment by Mister-Hope** [comment, marked as answer]
https://github.com/vuepress-theme-hope/vuepress-theme-hope/commit/799e9f83ccb9787870eb99790f00cd8b6da3c6d0
